appstudio-controller/controllers/appstudio.redhat.com/environment_controller_test.go

```diff
@@ -702,11 +852,58 @@ var _ = Describe("Environment controller tests", func() {
 			env = appstudioshared.Environment{}
 			err = reconciler.Get(ctx, req.NamespacedName, &env)
 			Expect(err).To(BeNil())
-			Expect(len(env.Status.Conditions)).To(Equal(1))
```

Comment:
Here (and elsewhere), I moved these into a new function to reduce code duplication between tests in this file.